spikeinterface/widgets/base.py

```diff
@@ -88,10 +96,7 @@ def define_widget_function_from_class(widget_class, name):
     @copy_signature(widget_class)
     def widget_func(*args, **kwargs):
         W = widget_class(*args, **kwargs)
-        # @alessio @jeremy @jeff
-        # function return the widget itself
-        # we could return something else if needed  as we discussed
-        return W
+        return W.plotter
```

Comment:
@samuelgarcia it makes more sense to return the plotter because that enables to interact with the backend (e.g. change title in mpl, etc...)